### PR TITLE
Email

### LINKED ISSUE #22: Sending recipe link over email

### PR DESCRIPTION
**Pull request by samanthaylin3**
Resolves #22 

**What were your changes**

- added a button to the recipe html and css for email
- added the function for the button to the js

**What specifically should be checked**

there should be a "Email Recipe" button on all the recipe pages and when you click on the button it should open your default email and populate the email for you. but the receiver is left blank which the user should enter. the subject is just the recipe title + "recipe." and the body is just a link to the recipe for now.

This is what it should look like (the button should match the next/previous buttons now):
Screen Shot 2021-12-03 at 1 23 39 AM

### CODE REVIEW

**Review by HairforceNiko — APPROVED** [review]
Good job.

File: webpages/recipe.html [hunk 1]
```diff
@@ -23,6 +23,12 @@ <h2 class="recipe-name">
             Chicken Tikka Masala
             <button class="favorite-heart">
               <img src="../assets/favorite.svg" />
+              <button class="favorite-heart">
```
Comment (HairforceNiko) [inline webpages/recipe.html:26]: Maybe you need to delete this button

Comment (samanthaylin3) [reply webpages/recipe.html:26]: okay, thanks!!

File: styles/recipe.css [hunk 2]
```diff
@@ -201,6 +201,11 @@ label {
   width: 86vw;
 }
 
+.email-recipe {
```
Comment (HairforceNiko) [inline styles/recipe.css:204]: Maybe need to add a margin-top: -10px;

**Comment by HairforceNiko** [comment]
You have to fix the CodeFactor issues before you merge your pull request.

File: styles/recipe.css (hunk 2 above)
Comment (samanthaylin3) [reply styles/recipe.css:204]: okay, ill add it

**Review by eekaireb — APPROVED** [review]

**Pull request merged**